ui/src/app/applications/components/application-deployment-history/application-deployment-history-details.tsx

```diff
@@ -43,36 +104,54 @@ export const ApplicationDeploymentHistoryDetails = ({app, info, index}: props) =
                         index={0}
                         versionId={recentDeployments[index].id}
                     />
-                    <button
-                        type='button'
-                        className='argo-button argo-button--base application-deployment-history__show-parameter-details'
-                        onClick={() => setShowParameterDetails(!showParameterDetails)}>
-                        {showParameterDetails ? 'Hide details' : 'Show details'}
-                    </button>
 
-                    {showParameterDetails && (
-                        <DataLoader
-                            input={{...recentDeployments[index].source, targetRevision: recentDeployments[index].revision, appName: app.metadata.name}}
-                            load={src => services.repos.appDetails(src, src.appName, app.spec.project, 0, recentDeployments[index].id)}>
-                            {(details: models.RepoAppDetails) => (
-                                <div>
-                                    <ApplicationParameters
-                                        application={{
-                                            ...app,
-                                            spec: {...app.spec, source: recentDeployments[index].source}
-                                        }}
-                                        details={details}
-                                    />
-                                </div>
-                            )}
-                        </DataLoader>
+                    {showParameterDetails ? (
+                        <div
+                            id={`'history-expanded'`}
+                            key={`'history-expanded'`}
+                            className={classNames('white-box')}
+                            style={{
+                                marginTop: singleSourceTopMargin,
```

Comment:
do you think we can move it to styles ? or it is generic practice in ArgoCD ui?

Reply:
It should be in the css file.  I had a variable there `singleSourceTopMargin`. I going to remove it and move these styles out.